apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.YCSB/Program.cs

```diff
@@ -55,6 +66,8 @@ internal class Program : YcsbRunner
 
         private static SpannerConnection s_connection;
         private static TimestampBound s_timestampbound;
+        private static SpannerClient s_rawclient;
```

Comment:
Nits: s_rawClient and s_rawSessions. (And ideally s_timestampBound above, although that's already in.)